adapters/openrtb_util.go

```diff
@@ -172,3 +174,29 @@ func copyFormats(sizes []openrtb2.Format) []openrtb2.Format {
 	}
 	return sizesCopy
 }
+
+func ExtractBidderParams(bidRequest *openrtb2.BidRequest) (map[string]json.RawMessage, error) {
+	if bidRequest == nil {
+		return nil, fmt.Errorf("error bidRequest should not be nil")
```

Comment:
Nitpick: There is no need to use `fmt.Errorf` here for a fixed string. Please use `errors.New` instead. Not that it matters much.. this condition will never be hit in practice since the adapter is never called with a nil request. I'm also fine if you just want to remove this check.

Reply:
Fixed.

Reply:
Github is still showing `fmt.Errorf()`?

Reply:
Fixed